certsubmitter/certsubmitter.go

```diff
@@ -95,7 +95,7 @@ func checkSCT(sct *ct.SignedCertificateTimestamp, l *ctlog.Log) []error {
 	// TODO(katjoyce): Add a way to classify errors as 'misbehaviour' vs
 	// 'unexpected behaviour worth noting'.
 	if sct.SCTVersion != ct.V1 {
-		errs = append(errs, &NotV1Error{Version: sct.SCTVersion})
+		errs = append(errs, &V1Error{Version: sct.SCTVersion})
```

Comment:
This name is also confusing.
Consider `SCTVersionError` with Got and Want fields?

Reply:
The struct nature of this might not be fully nessesary. Consider 
`var SCTVersionErr = errors.New("SCT Version was not 1.0")`

Reply:
For the errors that represent breaches of RFC6962/UA Policy, different types of errors will be classified and handled in different ways.  As such, it is quite useful to literally have different *types* of errors to represent them, because then decisions can be made based on what type of error was found.

The renaming is a great suggestion - I've also made the LogID one a little clearer using that idea too :)